packages/rpc-core/src/types/rpc-methods/getAccountInfo.d.ts

```diff
@@ -0,0 +1,92 @@
+import { Base58EncodedAddress } from '@solana/keys';
+import { IJsonRPCTransport } from '../../rpc';
+
+type Base64EncodedBytes = string & { readonly __base64EncodedBytes: unique symbol };
+type Base64EncodedZStdCompressedBytes = string & { readonly __base64EncodedZStdCompressedBytes: unique symbol };
+
+type Base64EncodedDataResponse = [Base64EncodedBytes, 'base64'];
+type Base64EncodedZStdCompressedDataResponse = [Base64EncodedZStdCompressedBytes, 'base64+zstd'];
+
+type GetAccountInfoApiResponseBase = Readonly<{
+    context: Readonly<{
+        slot: Slot;
+    }>;
+    value: Readonly<{
+        executable: boolean;
+        lamports: number; // TODO(solana-labs/solana/issues/30341) Represent as bigint
+        owner: Base64EncodedAddress;
+        rentEpoch: number; // TODO(solana-labs/solana/issues/30341) Represent as bigint
+        space: number; // TODO(solana-labs/solana/issues/30341) Represent as bigint
```

Comment:
Maybe I'm missing something, but in testing I didn't see `space` returned here. Here's a token account:

```
{
  "jsonrpc": "2.0",
  "result": {
    "context": {
      "apiVersion": "1.13.6",
      "slot": 181671824
    },
    "value": {
      "data": [
        "xvp6877brTo9ZfNqq8l0MbG75MLS9uDkfKYCA0UvXWEs07wIp7mh6Iz0YahgZDSe/Psf+oXDeVKNALXKNdFOFAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
        "base64"
      ],
      "executable": false,
      "lamports": 2039280,
      "owner": "TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA",
      "rentEpoch": 0
    }
  },
  "id": 1
}
```

Reply:
Haha. In the docs, this is called `size`, in the docs' example it's called `space`, in the current web3.js implementation it's absent, and in the server implementation it's `space` and it's an `Option<u64>`.

Reply:
🤔 https://github.com/solana-labs/solana/pull/28421

Reply:
Looks like this is only in v1.15.

Reply:
Ah gotcha